resources/views/layout/welcome.blade.php

```diff
@@ -0,0 +1,108 @@
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+
+
+ <link rel="stylesheet" type="text/css" href="css/index.css">
+
+
+ <!-- jQuery library -->
+ <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+ <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"/>
```

Comment:
Remove linking "bootstrap" here, as we will use vue and npm later. Please use:
```<link rel="stylesheet" href="css/app.css"/>```
Note: You were using bootstrap 3 from cdn here, laravel mix will use bootstrap 4. You can see that in 'package.json'. There will be some problems with the design to be fixed.